frontend/src/components/BattlePlans.svelte

```diff
@@ -62,11 +66,14 @@
         <div class="w-1/2 lg:w-3/4">
             <h3 class="text-2xl leading-tight font-bold">Plans</h3>
         </div>
-        <div class="w-1/2 lg:w-1/4 text-right">
+        <div class="w-1/2 lg:w-1/{allowJiraImport ? '2' : '4'} text-right">
```

Comment:
if you're adjusting this width you have to adjust the previous divs with as well to reflect

Reply:
Whats wrong? I'm not a frontend developer, so I not understand how change markup to do it perfect. Can you help me?

Reply:
Sure here's what you'd have to change on the sibling div to the one you've already adjusted

From
```html
<div class="w-1/2 lg:w-3/4">
```

To
```html
<div class="w-1/2 lg:w-{allowJiraImport ? '1/2' : '3/4'}">
```

Reply:
Thanks

Reply:
Fixed